contrib/kubernetes/rc/r1.yml

```diff
@@ -0,0 +1,43 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: etcd-1
+spec:
+  replicas: 1
+  selector:
+    app: etcd-1
+  template:
+    metadata:
+      labels:
+        app: etcd-1
+        service: etcd
+    spec:
+      containers:
+        - name: etcd
+          image: gcr.io/google_containers/etcd:2.0.12
```

Comment:
Yes also want to use quay.io?
On Dec 31, 2015 12:19 PM, "Xiang Li" <notifications@github.com> wrote:

> In contrib/kubernetes/rc/r1.yml
> <https://github.com/coreos/etcd/pull/4106#discussion_r48670042>:
>
> > +kind: ReplicationController
> > +metadata:
> > +  name: etcd-1
> > +spec:
> > +  replicas: 1
> > +  selector:
> > +    app: etcd-1
> > +  template:
> > +    metadata:
> > +      labels:
> > +        app: etcd-1
> > +        service: etcd
> > +    spec:
> > +      containers:
> > +        - name: etcd
> > +          image: gcr.io/google_containers/etcd:2.0.12
>
> can we use latest tag instead of 2.0.12 (2.0 is out of date)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/etcd/pull/4106/files#r48670042>.
>